torch/fx/_symbolic_trace.py

```diff
@@ -119,7 +119,28 @@ def _patch_function(fn: FunctionType, nargs: int) -> FunctionType:
     co = fn.__code__
     co_flags = co.co_flags & ~HAS_VARSTUFF
     co_args: tuple
-    if hasattr(co, "co_posonlyargcount"):
+    if hasattr(co, "co_qualname"):
```

Comment:
I guess this needs to be guarded for 3.11 to make the lint happy?

Reply:
Not sure if `mypy` is smart enough to detect it, but added ignore to the two violations